short-read-mngs/idseq_utils/idseq_utils/batch_run_helpers.py

```diff
@@ -21,7 +21,7 @@
 
 log = logging.getLogger(__name__)
 
-MAX_CHUNKS_IN_FLIGHT = 10
+MAX_CHUNKS_IN_FLIGHT = 30
```

Comment:
I actually forgot about this...

I actually think we should not have a max chunks in flight, and having it is kind of a disaster. We won't submit all the chunks at once we will wait for our first chunks to complete before submitting later chunks. However, in the meantime new chunks can enter the queue so we will now have the later chunks waiting on all of them keeping the alignment job running. This makes scaling way worse and if we hit timeouts and get even more retries on top of that we can have an explosion in waiting time. The max chunks in flight really doesn't protect us from going to too high of a scale because we can get too many alignment jobs running, it really only makes things worse.

I think making it 30 does this in effect because we don't have more than 30 chunks for any of our indexes. Would you mind adding a comment to point that out and we can re-visit removing this completely in the future?

Reply:
Done, yeah the intention was to effectively remove this bottleneck